.gitlab/build_lassen.yml

```diff
@@ -57,6 +57,12 @@ lassen-gcc_7_3_1-src:
     HOST_CONFIG: "lassen-blueos_3_ppc64le_ib_p9-${COMPILER}.cmake"
   extends: [.src_build_on_lassen]
 
+lassen-gcc_8_3_1_cuda-src:
```

Comment:
I think it would be good to also include the cuda version in the job labels. Do others agree?